### ISSUE TITLE
"actual" value is not printed if actual happened to be "undefined"

### ISSUE THREAD

**Issue by mo** [qunitjs/qunit#1177]
Using qunitjs 2.3.2, the test below fails to print the "actual" value, just because sum() happened to return undefined as the incorrect/buggy value. If it returned "wrong" instead, the "actual" is printed next to the "expected" so it's easier to see what went wrong. If a string value was expected and undefined was returned instead, then it would be nicer if qunit printed "actual: undefined" to make that clear.

```js
var isCurrentlyBuggy = true;

function sum(a, b) {
  if (isCurrentlyBuggy) {
    return undefined;
  }
  return a + b;
}

QUnit.only('my test', assert => {
  assert.equal(sum(1, 1), 2);
});
```

**Comment by trentmwillis** [comment]
This seems reasonable and likely an easy fix. Would you mind attempting a PR?

**Comment by mo** [comment]
Seems like it's printed from here:
https://github.com/js-reporters/js-reporters/blob/master/lib/reporters/TapReporter.js#L50

**Comment by mo** [comment]
PR here: https://github.com/js-reporters/js-reporters/pull/102

**Comment by trentmwillis** [comment]
I finally released `js-reporters@1.2.1` which should fix this issue.

**Comment by mo** [comment]
great, thanks
